package.json

```diff
@@ -2,9 +2,15 @@
   "name": "platform-chaos",
   "version": "1.0.0",
   "description": "A node sdk for building services capable of injecting chaos into PaaS offerings",
-  "main": "index.js",
+  "main": "dist/index.js",
   "scripts": {
-    "test": "mocha"
+    "lint": "npx tslint -c tslint.json 'src/**/*.ts'",
```

Comment:
i don't like that this has a path in it, let's capture that in tslint.json (which i believe is also implicit if you run `tslint .` or similar [implicit is preferred imho, as then if we move to a non json config file no additional change is needed])

Reply:
Per documentation https://palantir.github.io/tslint/usage/configuration/ I don't believe we can specify the path inside tslint.json.

However, `npx tslint 'src/**/*.ts'` does default to using `tslint.json`

Reply:
tsconfig, my bad